helpers/calendar/credentials.py

```diff
@@ -34,3 +34,11 @@ def set_api_credentials(self):
         service = build('calendar', 'v3', developerKey=api_key,
                         http=credentials.authorize(Http()))
         return service
+
+
+class CalendarApi:
```

Comment:
docstrings would be handy here too.

Reply:
I have also implemented this. @michael-basweti